android/res/xml/device_filter.xml

```diff
@@ -2,5 +2,6 @@
 <resources>
     <!-- Allow anything connected -->
     <usb-device />
+    <usb-accessory />
```

Comment:
Is there a way to limit this to the Taisync? The Android documentation says you should though I can't quite figure out what information you would provide that would _filter_ the Taisync only:

_In this case, the following resource file should be saved in res/xml/accessory_filter.xml and specifies that any accessory that has the corresponding model, manufacturer, and version should be filtered. The accessory sends these attributes the Android-powered device:_
```
<?xml version="1.0" encoding="utf-8"?>

<resources>
    <usb-accessory model="DemoKit" manufacturer="Google" version="1.0"/>
</resources>
```

Reply:
Yes indeed - I limited it to:
<usb-accessory model="android.usbaoa" manufacturer="taisync" version="1.0"/>